tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -230,23 +230,16 @@ def test_bump_expiry(self, monkeypatch, test_repo):
             "get_repository_settings",
             lambda *a, **kw: fake_settings,
         )
-        fake_time = datetime.datetime(2019, 6, 16, 9, 5, 1)
-        fake_datetime = pretend.stub(
-            now=pretend.call_recorder(lambda: fake_time)
-        )
-        monkeypatch.setattr(
-            "repository_service_tuf_worker.repository.datetime", fake_datetime
-        )
+        fake_datetime = mocked_datetime
```

Comment:
I don't see why you renamed `fake_datetime` to `mocked_datetime` everywhere.
I can understand if it's a single instance where a line goes above 79 characters if you use `mocked_datetime` instead and you decide to rename it, but why do you do it everywhere?
I think it's clear that if something is mocked then it's faked.

Reply:
I agree with you. Makes sense to use `mocked_datetime` directly. I just used `fake_datetime` because I got a reference that was already in the code, like in the function `test_sign_metadata_invalid_role_type` (current line 3145). So I'm going to change now, including this reference.